pulp_container/tests/functional/api/test_content_cache.py

```diff
@@ -186,6 +192,11 @@ def fetch_response_metadata(self, response):
         if DEFAULT_FILE_STORAGE in STANDARD_FILE_STORAGE_FRAMEWORKS:
             return response.status_code, response.headers.get("X-PULP-CACHE")
         else:
+            """Corner case for deprecated v2 schema1"""
```

Comment:
Which corner cases are we catching here? I thought that we were dealing only with manifest v2 schema 2 content inside this case.

Reply:
I don't remember now, but it was probably an error in test_05_different_headers.
I'm rerunning the tests without this block to confirm that the "corner case" was actually because of wrong headers in my tests.